cmd/exec.go

```diff
@@ -20,12 +20,15 @@ var (
 	assumeRoleTTL time.Duration
 )
 
+var profiles, _ = listProfiles()
```

Comment:
This pollutes the global namespace and doesn't check `err`. How about this?

```

var execCmd = &cobra.Command{
	Use:       "exec <profile> -- <command>",
	Short:     "exec will run the command specified with aws credentials set in the environment",
	RunE:      execRun,
	PreRun:    execPre,
	ValidArgs: listProfileNames(mustListProfiles()),
}

func mustListProfiles() lib.Profiles {
	profiles, err := listProfiles()
	if err != nil {
		log.Panicf("Failed to list profiles: %v", err)
	}
	return profiles
}
```

Reply:
Ok, this looks much better indeed!